composer.json

```diff
@@ -93,17 +89,13 @@
         "symfony/yaml": "^4.4 || ^5.3 || ^6.0",
         "vimeo/psalm": "^4.9.2"
     },
+    "conflict": {
+        "doctrine/doctrine-bundle": ">=3",
+        "sonata-project/doctrine-orm-admin-bundle": "<=4.0.0-alpha-1"
```

Comment:
IMHO we can remove this, because alpha releases could be buggy and we should not add conflicts for any buggy lib

Reply:
Removed, I think the same.